src/components/MyData/styles.ts

```diff
@@ -0,0 +1,59 @@
+import { StyleSheet } from 'react-native';
+import { VictoryThemeDefinition } from 'victory-core';
+import { createStyles } from '../BrandConfigProvider';
+
+type StyleExtractor<T> = T extends { style: unknown } ? T['style'] : never;
```

Comment:
This is to resolve the correct nested style typings when passing in style overrides via a prop, is that right? Pretty cool!

Reply:
Yeah exactly. I didn't confirm but I would bet that not all of the keys of `VictoryThemeDefinition` have a `style` property (i.e. `VictoryThemeDefinition.[key].style?`) and so I needed to add the conditional typing so that typescript could strongly type the style prop when extracting it only if it exists.